packages/bpk-component-content-cards/src/ContentCard/ContentCard.module.scss

```diff
@@ -0,0 +1,92 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+@import '~bpk-mixins/index';
+
+$bpk-spacing-v2: true;
+
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+
+  &::before {
+    content: '';
+    display: block;
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+
+  > :first-child {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.bpk-content-cards {
+  &__card-link {
+    color: inherit;
+    font: inherit;
+    text-decoration: inherit;
+  }
+
+  &__card-horizontal {
+    display: grid;
+    column-gap: bpk-spacing-xxl();
+    grid-template-columns: 50% 40%;
+    align-items: center;
+
+    @include bpk-breakpoint-mobile {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  &__card-horizontal &__image-aspect-ratio {
+    width: 100%;
+    height: 100%;
+
+    @include bpk-breakpoint-mobile {
+      max-height: $bpk-one-pixel-rem * 180;
+      margin-bottom: bpk-spacing-base();
+    }
+
+    @include bpk-breakpoint-above-tablet {
+      @include aspect-ratio(3, 2);
+    }
+  }
+
+  &__card-vertical &__image {
+    height: $bpk-one-pixel-rem * 460;
+    margin-bottom: bpk-spacing-base();
+
+    @include bpk-breakpoint-small-tablet {
+      width: 100%;
+      height: auto;
+      max-height: $bpk-one-pixel-rem * 260;
```

Comment:
Is there a way we can use other multiple of spacing tokens rather than just of 1 * x

Reply:
`($bpk-spacing-md + $bpk-spacing-md-v-2) * 10` ?